src/components/molecules/UiRating/UiRating.vue

```diff
@@ -189,7 +189,6 @@ const defaultProps = computed(() => {
   const iconDefault: IconAttrsProps['icon'] = 'star-outlined';
   const iconActive: IconAttrsProps['icon'] = 'star-filled';
   return {
-    translation: { stars: (index: number) => (`${index} stars`) },
```

Comment:
this is the solution for your issue, please use it to keep consistency in our library 
```
translation: {
  stars: (index: number) => (`${index} stars`),
  ...props.translation,
},
```